db/integrations.go

```diff
@@ -138,6 +148,16 @@ type IntegrationOpsgenie struct {
 	Incidents  bool   `json:"incidents"`
 }
 
+type IntegrationWebHook struct {
+	WebHookUrl         string `json:"webhook_url"`
+	CorrectResponse    string `json:"correct_response"`
```

Comment:
For example: telegram always answers with 200 and need check `ok` filed in response json.

Reply:
I've just received 401 status code using the wrong bot token.
```go
resp, err := http.Post(t.webhookUrl, "application/json", &data)
if err != nil {
    return err
}
defer resp.Body.Close()
klog.Infoln(resp.StatusCode, resp.Status)
```
```
I0222 19:32:21.399980   97830 webhook.go:88] 401 401 Unauthorized
```

Reply:
I checked too. And indeed it is. But I remember that a couple of years ago I received a 200 code when I made my bot.
Of course it would be easier and better without checking the content. But are you sure that all services will issue correct codes?